### PR TITLE
Add fixed point scaling config option

### PR DESCRIPTION
**Pull request by andrewclink**
This PR allows one to format currency that is stored in cents (i.e. 123 = $1.23). I called the option 'precision' but 'scaling' might have been a better idea; I'm open to suggestions. All tests passing.

### CODE REVIEW

File: bili.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   vue: {
     css: false,
     template: {
-      isProduction: true,
+      isProduction: false,
```
Comment (mazipan) [inline bili.config.js]: Why we turn `false` this part?

File: dist/vue-currency-filter.cjs.js [hunk 2]
```diff
@@ -0,0 +1,359 @@
+'use strict';
```
Comment (mazipan) [inline dist/vue-currency-filter.cjs.js]: Can you delete all files under `dist/` folder.
It should be in our `.gitignore`

File: package.json [hunk 3]
```diff
@@ -88,6 +88,7 @@
     "workbox-webpack-plugin": "^3.6.3"
   },
   "dependencies": {
-    "accounting": "0.4.1"
+    "accounting": "0.4.1",
+    "node": "^10.15.2"
```
Comment (mazipan) [inline package.json]: We should not put node as dependency

File: bili.config.js (hunk 1 above)
Comment (andrewclink) [reply bili.config.js]: Ahh, I was finishing up another project while packing for a conference. It looks like a few mistakes slipped through. I’ll review these, squash the commits and resubmit.

**Pull request closed without merging**